.gitignore

```diff
@@ -1,9 +1,14 @@
 .vscode/
+.idea/
 .nyc_output/
 .test_results/
 node_modules/
 coverage/
 
+**/package-lock.json
+
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+**/dist/
```

Comment:
stuff that's specific to a package, let's put it in the package's gitignore so the root doesn't need to be aware of specifics of a nested package